bu_passport/lib/pages/onboarding_page.dart

```diff
@@ -80,52 +90,56 @@ class _OnboardingPageState extends State<OnboardingPage> {
             children: _pages,
           ),
           Positioned(
-            bottom: 10,
-            left: 0,
-            right: 0,
-            child: Row(
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: List<Widget>.generate(
-                5, // Number of pages
-                (int index) {
-                  return AnimatedContainer(
-                    duration: Duration(milliseconds: 300),
-                    height: 10,
-                    width: _currentPage == index ? 30 : 10,
-                    margin: EdgeInsets.symmetric(horizontal: 5, vertical: 30),
-                    decoration: BoxDecoration(
-                      borderRadius: BorderRadius.circular(5),
-                      color: _currentPage == index ? Colors.blue : Colors.grey.withOpacity(0.5),
+              bottom: 10,
```

Comment:
Use dynamic measurements, instead of magic numbers

Reply:
Just made the change + some additional margins for the onboarding page. I think we are good to merge after your approval? The logo images have this weird blue line across it but I think Zihan can upload new logo photos to remove that after